pyravendb/tests/database_changes/test_database_changes.py

```diff
@@ -67,6 +67,12 @@ def test_for_document(self):
         self.assertEqual('Put', documents[0]["Type"])
         self.assertIsNotNone(documents[0]["ChangeVector"])
 
+    def test_for_max_document(self):
```

Comment:
why this test is in Database_Changes tests?

Reply:
Fixed.